frontend/src/components/reporting/ReportingContainer.cy.ts

```diff
@@ -15,19 +15,20 @@ describe('ReportingContainer.vue', () => {
     cy.fixture('ErrorLog.txt').then((errorLog) => {
       cy.fixture('ProgressLog.txt').then((progressLog) => {
         cy.fixture('YieldTable.csv').then((yieldTable) => {
-          cy.task('log', `Loaded ErrorLog: ${errorLog}`)
-          cy.task('log', `Loaded ProgressLog: ${progressLog}`)
-          cy.task('log', `Loaded YieldTable: ${yieldTable}`)
+          // logging
```

Comment:
If we don't need this logging anymore, you can probably remove the code rather than leave it in and commented out